Binumber.java

```diff
@@ -0,0 +1,37 @@
+public class Binumber {
+    private int number1;
+    private int number2;
+    int getNumber1(){
+       return this.number1;
+    }
+    int getNumber2(){
+        return this.number2;
+    }
+    Binumber(int number1 , int number2){
+        this.number1 = number1;
+        this.number2 = number2;
+    }
+    int add(){
+        int sumOfNumber = this.number1+ this.number2;
```

Comment:
please follow coding standard.
```suggestion
        int sumOfNumber = this.number1 + this.number2;
```